added myBooks and logout features

smilejin92 opened the pull request:
day4 hw

smilejin92 committed:
added myBooks and logout features

2woongjae commented on src/components/AddBookForm.jsx:
```diff
@@ -0,0 +1,58 @@
+
+  const handleAddForm = e => {
+    e.preventDefault();
+    axios
```
async - await 로 한번 해보세용~

2woongjae commented on src/pages/MyBooks.jsx:
```diff
@@ -0,0 +1,46 @@
+import Header from '../components/Header.jsx';
+import AddBookForm from '../components/AddBookForm.jsx';
+
+function MyBooks(props) {
```
`props => {token}` 으로 쓰는게 더 좋아보여요!

2woongjae commented on src/pages/MyBooks.jsx:
```diff
@@ -0,0 +1,46 @@
+  const [addBook, setAddBook] = useState(false);
+
+  useEffect(() => {
+    axios
```
이 안을 async - await 로 쓰는 법을 고민해보세용

2woongjae commented on src/pages/Signout.jsx:
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+export default function Signout() {
```
서버의 토큰도 지워야 합니다!

2woongjae reviewed (commented):
몇 가지 코멘트를 달아두었습니당 :)

smilejin92 committed:
1. converted handleAddForm function in components/AddBookForm.jsx to aync function.
2. destructured parameter props in pages/MyBooks.jsx
3. added useEffect in Signout.jsx to request deleting token as component
mounted

smilejin92 committed:
- Removed src/page/Signout.jsx
- Added `authorized` state in src/hocs/withAuth.js to check token in local storage

smilejin92 committed:
Removed imported modules in src/pages/Home.jsx that are no longer used.

2woongjae reviewed (commented):
수정한 코드 확인했습니다~
